doc/vim-go.txt

```diff
@@ -1589,6 +1588,23 @@ By default "snakecase" is used. Current values are: ["snakecase",
 >
       let g:go_addtags_transform = 'snakecase'
 <
+                                                        *'g:go_fold_enable'*
```

Comment:
I think we should add a line explaining that they get this when they also add `set foldmethod=syntax` to their vimrc